### DISCUSSION TITLE
An ability to always use gameplay (skin) cursor instead of UI version

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#14545] [Ideas] [answered]
As it's in osu!stable i want to decide which cursor i will use: UI or skin cursor. Will such a line be added to the skin settings section?

**Comment by peppy** [comment, marked as answer]
The UI cursor will probably be skinnable eventually, yes.

**Comment by Saperxd** [comment]
I think it would be simpler for now to add the option to disable it and replace it with the current gameplay cursor instead.
Personally I find it very confusing when I have to click with the corner instead of the centre which is the reason why I would want that option.

**Reply by peppy** [reply]
Thanks for the feedback. I agree to a certain extent. Maybe we'll iterate on the UI cursor to have it be more circular.

@arflyte maybe something to keep in the back of your mind?
